code/modules/atmospherics/machinery/components/fusion/hfr_main_processes.dm

```diff
@@ -516,16 +504,17 @@
 			else
 				filtering = FALSE
 		if(filtering && moderator_internal.gases[filter_type])
-			var/datum/gas_mixture/removed = moderator_internal.remove_specific(filter_type, 20 * delta_time)
+			var/datum/gas_mixture/removed = moderator_internal.remove_specific(filter_type, 100 * delta_time)
```

Comment:
?

Reply:
i increased the filtering because it was way too slow and the HFR usually ended up exploding because the players couldn't remove gases fast enough

Reply:
I wish you'd document these things, doesn't need to go in the changelog and you don't need to be exhaustive, but just, somewhere

Reply:
ye will do, i usually make a lot of changes and this seemed minor, but i agree i'll put more attention to it